migration/migration-export.sh

```
@@ -0,0 +1,145 @@
+
+# Step 9: Add .gitignore
+cat > $EXPORT_DIR/.gitignore << 'EOF'
+node_modules
+coverage
+.DS_Store
+.env
+*.log
+EOF
+
+# Step 10: Initialize git repository in the new project
+cd $EXPORT_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)